system/ui/lib/application.py

```diff
@@ -138,4 +142,11 @@ def _monitor_fps(self):
       os._exit(1)
 
 
-gui_app = GuiApplication(2160, 1080)
+def _init_gui_app():
+  if TICI:
+    return GuiApplication(2160, 1080)
+  else:
+    return GuiApplication(0, 0)
+
+
+gui_app = _init_gui_app()
```

Comment:
With my monitor resolution, the UI is just full black because of the current scaling